vgen/src/main/java/io/gapi/vgen/csharp/CSharpContextCommon.java

```diff
@@ -92,18 +100,21 @@ public String underscoresToCamelCase(
   // This function is necessary to provide a static entry point for the same-named
   // member function.
   public static String s_underscoresToCamelCase(
-      String input, boolean capNextLetter, boolean preservePeriod) {
+      String input, boolean capNextLetter, boolean preservePeriod, boolean forceLower) {
```

Comment:
Good point, yes I'll change this to lower-case the input first, and revert the change to this function. Not currently sure about the state of its portyness.
And I agree about unit tests for functions like this. ...in a follow-up PR, etc, etc...